src/services/Models.service.tsx

```diff
@@ -208,6 +208,9 @@ export const useEmotion = (nodes: ObjectMap['nodes'], emotion?: Emotion) => {
 export const useGltfLoader = (source: Blob | string): GLTF =>
   suspend(async () => {
     const loader = new GLTFLoader();
+    const dracoLoader = new DRACOLoader();
```

Comment:
should we duplicate this logic for the animations? can animations be draco encoded? src/services/Animation.service.ts

Reply:
Animations aren't meshes, so that's no a use case for Animation.service.